src/lift.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+module.exports = function(Promise) {
+
+Promise.lift = function ( fn ) {
+    var that = this;
+    return function() {
+        var $_len = arguments.length;
+        var args = new Array($_len+1);
+        for(var $_i = 0; $_i < $_len; ++$_i) {args[$_i] = arguments[$_i];}
+        args[$_len] = fn;
+        return Promise.join.apply( that, args );
```

Comment:
No, unless we can guarantee that `this`==`Bluebird` when the function is called. I keep `this` in `that` to make sure the original function is called with the same `this` binding than the lifted function.